boards/x230-maximized/x230-maximized.config

```diff
@@ -78,5 +78,14 @@ export CONFIG_FLASHROM_OPTIONS="--force --noverify-all -p internal"
 #  - blobs/xx30/extract.sh
 #     To extract from backuped 8M (bottom SPI) ME binary, GBE and IFD blobs.
 
+# Make the Coreboot build depend on the following 3rd party blobs:
+$(build)/coreboot-$(CONFIG_COREBOOT_VERSION)/$(BOARD)/.build: \
+    $(pwd)/blobs/xx30/me.bin
+
+
+$(pwd)/blobs/xx30/me.bin:
+	COREBOOT_DIR="$(build)/$(coreboot_base_dir)" \
+		$(pwd)/blobs/xx30/download_clean_me.sh $(pwd)/blobs/xx30
+
```

Comment:
I think this is duplicated 17 times for xx30.  Could you please move this snippet to `targets/xx30_me.mk` and then add `xx30_me` to BOARD_TARGETS for each board rather than duplicating this code?  Please do that for xx20 also.  Maybe the vbios bits too, but those aren't as much code or duplicated as many times.

You will also need a tweak in Makefile that I left out, change `include targets/$(BOARD_TARGETS).mk` to `include $(foreach $(BOARD_TARGETS),TARGET,targets/$(TARGET).mk)`.  (I meant for BOARD_TARGETS to allow multiple entries but evidently left out the foreach.)

Reply:
@JonathonHall-Purism fixed under https://github.com/linuxboot/heads/pull/1570/commits/f2079dbe4435f9130242996057fa51b1a534e70b https://github.com/linuxboot/heads/pull/1570/commits/d7c2bda11200e9f9caed10d5f030ccfcd083b30b and https://github.com/linuxboot/heads/pull/1570/commits/ad2a597a39d623a979fc08fb9731a470be27557c

TODO in next PR VBIOS download script needs way more work for currently w530/T530 unclaimed tested boards

Reply:
@JonathonHall-Purism 

> You will also need a tweak in Makefile that I left out, change include targets/$(BOARD_TARGETS).mk to include $(foreach $(BOARD_TARGETS),TARGET,targets/$(TARGET).mk). (I meant for BOARD_TARGETS to allow multiple entries but evidently left out the foreach.)

Added per Makefile changes under https://github.com/linuxboot/heads/pull/1570/commits/77f9933538fe00d9cbf1ed0887c4758f9b10bb77